config/prod.webpack.plugins.js

```diff
@@ -0,0 +1,14 @@
+/* global require, module */
+
+const { plugins } = require('./base.webpack.plugins');
+const webpack = require('webpack');
+
+/**
+ * sets release to insightsbeta
+ */
+const Release = new webpack.DefinePlugin({
+    RELEASE: JSON.stringify('insights')
```

Comment:
this should use `config.paths.publicPath` as the value so that both `insights` and `insightsbeta` will be supported.